test/configCases/plugins/define-plugin/index.js

```diff
@@ -105,4 +105,12 @@ it("should evaluate composed expressions (issue 5100)", function() {
 	} else {
 		require("fail");
 	}
-})
+});
+
+it("should follow renamings in var (issue 5215)", function() {
+	var _process$env = process.env,
```

Comment:
@sokra in your test environment, do you have a global process object defined? In a browser environment, attempting to access `process.env` throws the exception mentioned in #5215